src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -40,6 +40,11 @@ def add_strategy(self, strategy_tuple):
     """Add a strategy into our existing strategy pool."""
     self.strategy_names.add(strategy_tuple.name)
 
+  def remove_strategy(self, strategy_tuple):
```

Comment:
Discussed offline. This could end up getting a bit brittle depending on the order of how things are modified, and we don't necessarily expect everything enabled in the pool to be added to fuzzing_strategies if it causes issues.

Reply:
Done!